YIF.Core.Service/Concrete/Services/SuperAdminService.cs

```diff
@@ -405,5 +405,25 @@ public async Task<ResponseApiModel<IEnumerable<IoEModeratorsForSuperAdminRespons
                 Success = true
             };
         }
+
+        public async Task<ResponseApiModel<DescriptionResponseApiModel>> ChooseIoEAdminFromModerators(IoEAdminAddFromModeratorsApiModel ioEAdminAddFromModeratorsApiModel)
+        {
+            var result = new ResponseApiModel<DescriptionResponseApiModel>();
+            var moderator = await _ioEModeratorRepository.GetByUserId(ioEAdminAddFromModeratorsApiModel.UserId);
+            var dbUser = _userManager.Users.FirstOrDefault(x=>x.Id == moderator.User.Id);
```

Comment:
Shouldn't it be the method from repository?

Reply:
changed

Reply:
ok, thanks!!!